### ISSUE TITLE
EDD Blocks: Load base elements styles by default

### ISSUE THREAD

**Issue by dashkevych** [awesomemotive/easy-digital-downloads#9597]
It would be great if blocks loaded base styles by default to avoid causing style inconsistency on a website.

WordPress theme is responsible for appearance of a website, while a WordPress plugin is responsible for additional functionality on a website.

When the plugin adds its own opinionated styles to a block, it might break an overall design of a website.

For instance, let's take a button element from the EDD Login Form block.

It looks like the button element comes with an animation effect:

```css
.edd-button-secondary,
.edd-submit {
  transition: all .2s ease-in-out;
}

.edd-button-secondary:active,
.edd-button-secondary:hover,
.edd-submit:active,
.edd-submit:hover {
  transform: translateY(-1px);
}
```
It means that if a website does not have animation for buttons by default then only EDD buttons will have an animation.  As a result, it will cause style inconsistency.

And buttons from EDD will not feel very native to overall design concept of a website.

The same thing applies to paddings in input fields, etc.

It would be great if EDD had style options similar to WPForms (No styles, base styles, base + form styles) but keeping base styles as default option.

Thank you for your consideration :)

**Comment by Tropicalista** [comment]
I agree.

I want also add that a `wp-element-button` class to every button will make style consistent with theme. Also not every block has a filter to add this class. 

It would be also useful if edd blocks use default block styles. For example there's no need to add most of style that can be simply accomplished with the Gutenberg options.

An option to disable every block style is needed.

The blocks repository is private, otherwise I wil be more than happy to contribute.

**Comment by cklosowski** [comment]
We have resolved this with EDD 3.2.0
